components/Footer.jsx

```diff
@@ -43,14 +46,16 @@ export default function Footer() {
             />
           </a>
         </Link>
-        <h3 className=" text-xs">
-          <p>The Local cost of living dashboard is run by Economy:</p>
+        <h2 className=" text-xs">
```

Comment:
I think this should all just be a <p> tag, with the first sentence not distinguished by semi-bold. No need for a title here, I think